### PR TITLE
Update go-bouncer to build arm64 binaries for macOS and Linux

### PR DESCRIPTION
**Pull request by subecho**
This also updates the goreleaser install process to use "go install" and
will use the latest version available for goreleaser.

Signed-off-by: Dustin Schoenbrun <dustin.schoenbrun@anchore.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,7 +56,7 @@ bootstrap: ## Download and install all project dependencies (+ prep tooling in t
 		tar -xzvf pkger_0.17.0_Linux_x86_64.tar.gz pkger && \
 		mv pkger $(BIN)
 	# install goreleaser
-	curl -sfL https://install.goreleaser.com/github.com/goreleaser/goreleaser.sh | sh -s -- -b $(BIN) v0.138.0
+	GOBIN=$(BIN) go install github.com/goreleaser/goreleaser@latest
```
Comment (wagoodman) [inline Makefile]: we should be pinning to a specific version of goreleaser and not `latest` :
```
GOBIN=/home/runner/work/go-bouncer/go-bouncer/.tmp/bin go install github.com/goreleaser/goreleaser@latest
can't load package: package github.com/goreleaser/goreleaser@latest: can only use path@version syntax with 'go get'
```

Comment (subecho) [reply Makefile]: Whelp, Weston wins that bet 🤣 . I'll wire him £10 and pin it to the latest version, `v1.3.1` 👍

**Comment by subecho** [comment]
Got the pipeline green minus tests for the ARM artifacts which apparently don't have runners in GitHub. Should be ready to go now 👍

**Comment by wagoodman** [comment]
I went ahead and removed windows support for now to simplify this some

**Pull request merged**